functions/node8/index.js

```diff
@@ -20,22 +20,6 @@ const fetch = require('node-fetch');
 // [END functions_background_promise_node8]
 // [END functions_background_async]
 
-/**
- * HTTP Cloud Function (same signature as other Node runtimes)
```

Comment:
@fhinkel should we leave this file alone for now? (We'll need to migrate existing samples to Node 8 once we have a dedicated branch for Node 6, and can delete this then.)

Reply:
Sounds good. I'll revert those changes.

Reply:
Done